canonical-web-and-design/maas-ui
# fix(legacy): fix tags input overlaying other fields in machine config form

## Done

- Fixed tags input overlaying fields above in machine config form

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Make sure you're running latest Chrome
- Go to a machine's config form in legacy
- Add a bunch of tags to the tags input and check that the input extends downward instead of upward

## Fixes

Fixes #1824
